<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QWidget" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gravity field App</string>
  </property>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Start/Pause</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Add...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="del_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="preset_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Use presets...</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="euler_check">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Also show
euler method</string>
   </property>
  </widget>
  <widget class="QListWidget" name="part_list">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="error_hint">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="edit_button">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
